content/en/docs/components/model-registry/getting-started.md

```diff
@@ -7,7 +7,7 @@ weight = 20
 This guide shows how to get started with Model Registry and run a few examples using the
 command line or Python clients.
 
-At this time, the Model Registry does not include a web-based User Interface (UI), therefore this documentation focuses on backend services and APIs.
+<!-- TODO: Check with MR team to see if we include screenshots for a UI "get started" -->
```

Comment:
@lucferbux, we can do this in a FUP PR, but as the UI is an important piece of Model Registry, I would suggest figuring out how to mention the UI in this section.